docs/userGuide/plugins/web3Form.md

```diff
@@ -0,0 +1,94 @@
+### Plugin: Web3Forms 
+
+This plugin allows you to create forms whose response will be sent directly to your email, using the [Web3Form](https://web3forms.com/) API. 
+<box type="warning" seamless>
+Emails from Web3Forms often end up in spam / junk mail.
+</box>
+
+<box type="warning" seamless>
+The free plan from Web3Form only allows 250 submissions per month.
+</box>
+
+To set it up, get an access key from [Web3Forms](https://web3forms.com/). Then add `web3Form` to your site's plugin, and add the `accessKey` parameter via the `pluginsContext`.
```

Comment:
We should mention whether the access key is ok to be exposed or at least make known that it is exposed, given that it's a working form without a backend. I tried looking at the web3forms docs and it doesn't seem to mention it at all. Perhaps it's due to the fact that the project seems really new...... Nonetheless, we should do a bit of research and put a note about this in our docs, explaining:
- whether this key is exposed to public
- whether such exposure is risky and has any consequences

Reply:
The key will be exposed to the public since users can view the API call to web3forms and see the access key 
ie Developer tools -> network -> payload 
![image](https://user-images.githubusercontent.com/77204346/223029160-941a947e-8c2c-422b-ab6c-07e24f1653f2.png)
It is [mentioned](https://docs.web3forms.com/getting-started/installation#:~:text=%3C!%2D%2D%20REQUIRED%3A%20Your%20Access%20key%20here.%20Don%27t%20worry%20this%20can%20be%20public%20%2D%2D%3E) that its ok for the access key to be public. I emailed them to ask why and they said 

> Yes, it can be public.
> 
> The access key is an alias to your email. This is same as revealing email in public but one step harder. 
> 
> The people who get the access key can only used to email you. Nothing else. We also have spam protection + reCaptcha & domain protection in the Pro plan.

The problem with exposing the key is that other forms can use it and attackers might use it to DOS especially since only 250 submission are allowed per month as mentioned by @jovyntls. 

I will add the above to the documentation!

Reply:
Is this in response to the question above:
![image](https://user-images.githubusercontent.com/41845017/226260242-0d82b181-a3c5-48b3-8538-0170628fc3ce.png)

?

Can we re-word it such that it says something along the line of
- generally ok to be public as shared by the team behind it (the reason you gave here)
- however, some potential problem is xxxx

This is because, at the moment, I'm not sure what to make of the warning after reading it...-> it seems to suggest that I shouldn't expose the access key given all the negatives

Reply:
Thanks for the comments @tlylt, updated in [eef1b38](https://github.com/MarkBind/markbind/pull/2191/commits/eef1b38665d42a54440dfd7bb1d401def31fef98)